handler/src/main/java/io/netty/handler/ssl/SslUtils.java

```diff
@@ -71,6 +74,54 @@
      */
     static final int NOT_ENCRYPTED = -2;
 
+    static final String[] DEFAULT_CIPHER_SUITES = new String[] {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this array "private". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1873)